demo/sdk/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.10)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQURIED ON)
+
+project(play_cpp_sdk VERSION 0.0.1)
```

Comment:
should the version be taken from a git tag or what's the common convention in cmake?